src/bloqade/codegen/common/is_hyperfine.py

```diff
@@ -0,0 +1,28 @@
+from typing import Any
+import bloqade.ir.analog_circuit as analog_circuit
+import bloqade.ir.control.sequence as sequence
+from bloqade.ir.visitor.analog_circuit import AnalogCircuitVisitor
+
+
+class IsHyperfineSequence(AnalogCircuitVisitor):
+    def __init__(self):
+        self.is_hyperfine = False
+
+    def visit_analog_circuit(self, ast: analog_circuit.AnalogCircuit) -> Any:
+        self.visit(ast.sequence)
+
+    def visit_append_sequence(self, ast: sequence.Append) -> Any:
+        list(map(self.visit, ast.sequences))
```

Comment:
coverage complains these visitors are not tested

Reply:
we don't currently support appended sequences in the builder so I can just comment this out